ml_git/commands/utils.py

```diff
@@ -4,23 +4,24 @@
 """
 
 from ml_git.config import config_load
+from ml_git.constants import EntityType
 from ml_git.log import set_level
 from ml_git.repository import Repository
 
-DATASET = 'dataset'
-LABELS = 'labels'
-MODEL = 'model'
+DATASETS = EntityType.DATASETS.value
+LABELS = EntityType.LABELS.value
+MODELS = EntityType.MODELS.value
 PROJECT = 'project'
 
 
-def init_repository(entity_type='dataset'):
+def init_repository(entity_type='datasets'):
```

Comment:
```suggestion
def init_repository(entity_type=DATASETS):
```
Perhaps it's better to get rid of any hardcoded value as we have the opportunity